src/css/components/_progress.scss

```diff
@@ -86,12 +86,12 @@
 .video-js .vjs-play-progress:after {
   display: none;
   position: absolute;
-  top: -2.4em;
+  top: -3.5em;
   right: -1.5em;
   font-size: 0.9em;
   color: #000;
   content: attr(data-current-time);
-  padding: 0.2em 0.5em;
+  padding: 6px 8px 8px 8px;
```

Comment:
So, after looking careful at it with @incompl, the computed style does actually lie here. What was causing as issue is the slight overlap of the control bar which caused it to look wrong to me. This is fixed by the latest commit to the PR.